msvc-full-features/Cataclysm-lib-vcpkg-static.vcxproj

```
@@ -122,7 +122,7 @@
       <PreprocessorDefinitions>_CONSOLE;SDL_SOUND;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <PreBuildEvent>
-      <Command>..\msvc-full-features\prebuild.cmd</Command>
+      <Command>python ..\build-scripts\version.py</Command>

```

We can make this optional.
```suggestion
      <Command>python ..\build-scripts\version.py || cd . > NUL</Command>

```